src/client/gitlab.ts

```diff
@@ -107,13 +107,15 @@ export const getGroupsData = async (
 
   const queryParams = queryParamsGenerator(params);
 
+  console.log(`Calling gitlab to get groups. Query params: ${queryParams}`);
   const { data } = await callGitlab(`/api/v4/groups?${queryParams}`, groupAccessToken);
 
   return data;
 };
 
 export const registerGroupWebhook = async (payload: RegisterWebhookPayload): Promise<number> => {
   const { groupId, token: groupToken, url, signature } = payload;
+  console.log(`Calling gitlab to register webhook`);
```

Comment:
Maybe instead of logging at each method, `callGitlab` can take the name of the calling function and log it for all requests?